pineappl_py/tests/test_boc.py

```diff
@@ -10,11 +11,53 @@ def test_init(self):
 
 
 class TestKinematics:
-    def test_init(self):
-        kin = Kinematics.Scale(0)
-        scale_func = ScaleFuncForm.Scale(0)
-        assert isinstance(kin, Kinematics)
-        assert isinstance(scale_func, ScaleFuncForm)
+    @pytest.mark.parametrize(
+        "kintype, argument, expected_type",
```

Comment:
the third element seems to be always the same - I guess we can hardcode it, no? same for `ScaleFuncForm`

Reply:
Done in [e7c2e88](https://github.com/NNPDF/pineappl/pull/299/commits/e7c2e880c776ed620668ef8b2954cc9853961db2).